ChangeLog.d/9618.txt

```diff
@@ -0,0 +1,12 @@
+Removals
+   * Previously deprecated build symbol `MBEDTLS_ENTROPY_HARDWARE_ALT` is
+     removed.
+
+Features
+   * The new symbol `MBEDTLS_ENTROPY_HARDWARE_POLL` is added to allow the
```

Comment:
The changelog entries should probably mention the function names as well as the option names, for easier searching.